platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -122,6 +122,13 @@ typedef struct {
 	uint32_t headroom;
 	uint32_t tailroom;
 
+	/* Fields used to support packet references */
+	uint32_t unshared_len;
+	struct odp_packet_hdr_t *ref_hdr;
+	uint32_t ref_offset;
+	uint32_t ref_len;
+	odp_atomic_u32_t ref_count;
```

Comment:
I'll post a v3 with some comments on these if that would be helpful. I was just being consistent with the rest of the fields, which are similarly lightly documented.